.github/workflows/integrate.yaml

```diff
@@ -38,24 +38,26 @@ jobs:
 
   unit:
     name: Unit Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Install dependencies
         run: pip install tox
       - run: tox -vve unit
 
   integration:
     name: Integration Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Setup operator environment
-      uses: charmed-kubernetes/actions-operator@main
+      uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```

Comment:
why do we use charmed-kubernetes in [argo-operators](https://github.com/canonical/argo-operators) and claudiubelu in another?

either align or add a comment

Reply:
Ah, we missed this repo. Will update in a [separate PR](https://github.com/canonical/metacontroller-operator/pull/51)

Reply:
ok, other PR is approved, then wait for it to merge and this one will be ready then